README.md

```diff
@@ -1,4 +1,12 @@
-# ChatGPT Retrieval Plugin
+# ChatGPT Retrieval Plugin, Hosted on Render
+
+Fork this repo and click the button below to deploy your ChatGPT Retrieval Plugin server. You will be prompted to enter your OpenAI API key. The bearer token for your retrieval plugin server will be generated automatically. You can find it in the "Environments" tab on the Render dashboard page for the server.
```

Comment:
maybe link https://dashboard.render.com/

Reply:
It's probably also worth noting that this:
* uses weaviate as the vector store
* uses starter plans for both the server and db, which cost money, anything you think is important about sizing for performance, and about the free option

Reply:
Good call. Let me know what you think of the additions.

Reply:
This looks better. Two suggestions (that Anurag or Meagan can also review):
* It might also be worth adding something about the possibility of free plans and limitations
* Since prices can change, you could also direct to the pricing page (e.g. as of April 2023, this costs $15/month, for latest check render.com/pricing).

Reply:
On the first point I did add this:
> The FastAPI server is compatible with [Render's free plan](https://render.com/docs/free) but the Weaviate database is not because it has a disk.

Should I say more? Once we move to postgres the whole setup should be doable on the free tier.

Good call on pricing changes. Will add that.